pangeo_forge_recipes/transforms.py

```diff
@@ -399,4 +420,10 @@ def expand(self, datasets: beam.PCollection) -> beam.PCollection:
         target_store = schema | PrepareZarrTarget(
             target=self.get_full_target(), target_chunks=self.target_chunks
         )
-        return rechunked_datasets | StoreDatasetFragments(target_store=target_store)
+
+        stored_fragments = rechunked_datasets | StoreDatasetFragments(target_store=target_store)
+        return (
+            stored_fragments
+            if not self.consolidate_coords
+            else stored_fragments | ConsolidateDimensionCoordinates(target_store=target_store)
+        )
```

Comment:
```suggestion
        rechunked_datasets | StoreDatasetFragments(target_store=target_store)
        if self.consolidate_coords:
            ConsolidateDimensionCoordinates(target_store=target_store)
        return target_store
```

@norlandrhagen, following merge of https://github.com/pangeo-forge/pangeo-forge-recipes/pull/562, we now return `target_store` from `StoreToZarr`. I also realized that `StoreDatasetFragments` returns `None`, so I don't think we need to assign it to a variable. This suggestion incorporates these changes.